### DISCUSSION TITLE
Inability to build INET 4.5 in omnet++ 6.0.3

### DISCUSSION THREAD

**Discussion by anoikob** [inet-framework/inet#992] [General]
Greetings.
I have successfully installed omnet++ 6.0.3 and got it to work on Ubuntu 24.01.1 LTS (VM on Virtualbox, that runs on Win10). I have not touched its folder and contents, aside from sourcing the setenv file, configuring and making. I tried to install the latest stable INET version, through omnet, by navigating to Help > Install Simulation Models... > INET 4.5.3, but I've run into this problem, making the project unable to be built: 
![image](https://github.com/user-attachments/assets/071d2165-42e2-42a7-b5c5-179511d91a00)

Any ideas? The error messages are talking about missing files, but aside from the actions I described above, I have not dealt with the files in omnet. I just installed it, it worked, and then I tried installing INET. Any ideas?

Edit: I should probably mention that I got omnet AND inet to work on my windows PC (same versions), but I wanna do it on Linux too.

**Comment by torokati44** [comment]
What command have you used to build OMNeT++?
Just a guess: Did you perhaps specify `MODE=release`? If so, "debug mode" libraries weren't built, and as a result, simulation model libraries can't be built that way either, as the "MODEs" have to match. To remedy this, you can run the `make` command again, this time with `MODE=debug`.

Does the file `/home/alucard/omnetpp/omnetpp603/lib/liboppnedxml_dbg.so` exist? How about `liboppnedxml.so` (without the `_dbg` suffix) at the same location?

**Reply by anoikob** [reply]
Hello and thanks for the answer! I used just "make", and yes, they both exist in the same location. Is one for debugging?

![image](https://github.com/user-attachments/assets/33852705-4611-4a4e-a589-d9b7b5a78a88)

**Comment by anoikob** [comment]
I rebuilt omnetpp with MODE=debug , deleted every trace of inet because it still existed for some reason even though it did not show up in the projects menu, redownloaded it from the omnetpp IDE, and it now works
